meson.build

```diff
@@ -175,6 +175,10 @@ endif
 if cc.has_header('linux/if_xdp.h')
     if xdp_dep.found() and cc.has_header('xdp/xsk.h')
         if bpf_dep.found() and cc.has_header('bpf/bpf.h')
+            bpf_8_dep = dependency('libbpf', version : '>=0.8.0', required: false, method: 'pkg-config')
+            if bpf_8_dep.found()
+                cne_conf.set10('HAS_LIBBPF_8', true)
```

Comment:
Need to set cne_conf.set10('HAS_LIBBPF_8', false) around line 168.

Reply:
yeah makes sense :( doh

Reply:
One other item we can think about. We can remove HAS_LIBXDP and HAS_LIBXDP_8 defines then use the USE_LIBXDP and USE_LIBBPF_8 directly.

Remove the two ifdefs in cne_common.h for HAS_LIBXDP and HAS_LIBBPF_8 then change the meson.build the HAS_LIBXDP and HAS_LIBBPF_8 to USE_LIBXDP and USE_LIBBPF_8 to true or false. These will then be places in the cne_build_config.h file.

Then change the HAS_LIBXDP and HAS_LIBBPF_8 in the rest of the code to use USE_LIBXDP and USE_LIBBPF_8.

Does this work and is I hope cleaner. I can also make these changes after you do the one line change from above as this would be a different PR.

Reply:
no worries I can have a look. I think that's the way I originally tried it and when I started regression testing ran into some issues - I didn't debug as I thought the precedent was: [CAN_USE_XSK_UMEM_SHARED](https://github.com/CloudNativeDataPlane/cndp/blob/main/lib/include/cne_common.h#L52)... but I can dig in and see where I was going wrong before :)